src/index.js

```diff
@@ -89,6 +89,41 @@ kalamata.expose = function(model, _opts_) {
             var beforeResult = runHooks(beforeHooks.getCollection, [req, res, mod]);
             if(res.headersSent) return;
 
+            // If we send page or page_size, we want to limit the query
+            if(req.query.page || req.query.page_size) {
+                // By default, we return the first page with 100 items
+                const { page = 1, page_size = 100 } = req.query;
+                const page_number = parseInt(page, 10);
+                const page_size_number = parseInt(page_size, 10);
+
+                const { total_items, total_pages } = await mod.count('id').then(total_items => ({total_items, total_pages: Math.ceil(total_items/page_size_number)}))
```

Comment:
Can we split this objects onto newlines so it's easier to browse and read by other contributors?

```
mod.count('id').then(total_items => ({
    total_items,
    total_pages: Math.ceil(total_items / page_size_number)
}))
```